File: Persistencia01/app/src/main/java/com/pnultimedia/persistencia01/MainActivity.java
package com.pnultimedia.persistencia01;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private EditText editText1;
    private EditText editText2;
    private TextView textView1;
    private TextView textView2;
    private Button btnGuardar;
    private Button btnRecuperar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editText1 = findViewById(R.id.editText1);
        editText2 = findViewById(R.id.editText2);
        textView1 = findViewById(R.id.textView1);
        textView2 = findViewById(R.id.textView2);
        btnGuardar = findViewById(R.id.btn_guardar);
        btnRecuperar = findViewById(R.id.btn_recuperar);

        final SharedPreferences sPref = getSharedPreferences("persistencia", Context.MODE_PRIVATE);
        final SharedPreferences.Editor editor = sPref.edit();

    }
}

File: MiPrimeraApp/app/src/main/java/com/example/miprimeraapp/MainActivity.kt
package com.example.miprimeraapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {

        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Acceso a los elementos del layout.
        val textView = findViewById<TextView>(R.id.holaMundo1)
        textView.text = "Texto desde codigo."

        // Otra de forma de acceder (solo Kotlin)
        holaMundo2.text = "ADIOS MUNDO!!"


    }

}
